### DISCUSSION TITLE
Disabling entries in <musiclibrary> section of advancedsettings.xml

### DISCUSSION THREAD

**Discussion by azido** [croneter/PlexKodiConnect#1843] [Q&A] [answered]
Hey there,

even if there is no music library at all in plex, on every start plexkodiconnect checks and writes    backgroundupdate>true</backgroundupdate at musiclibraries portion of advancedsettings.xml of kodi.

I want to manage my music database inside kodi, not plex. Can you disable this behaviour above? Thing is, i can't see music library updating with this setting on, and i did not find a way to disable this setting.

**Comment by croneter** [comment, marked as answer]
See #1846
